articles/user-guide/libraries/standard/data-structures.md

```diff
@@ -174,52 +176,59 @@ $$
 $$
 \begin{align}
     i\frac{d \ket{\psi(t)}}{d t} & = H \ket{\psi(t)},
 \end{align}
 $$
-with a Hermitian matrix $H$, known as the Hamiltonian, as the generator of motion. Given an initial state $\ket{\psi(0)}$ at time $t=0$, the formal solution to this equation at time $t$ may be, in principle, written
+with a Hermitian matrix $H$, known as the [Hamiltonian](xref:microsoft.quantum.chemistry.concepts.quantumdynamics), as the generator of motion. Given an initial state $\ket{\psi(0)}$ at time $t=0$, the formal solution to this equation at time $t$ may be, in principle, written
 $$
 \begin{align}
     \ket{\psi(t)} = U(t)\ket{\psi(0)},
 \end{align}
 $$
-where the matrix exponential $U(t)=e^{-i H t}$ is known as the unitary time-evolution operator. Though we focus on generators of this form in the following, we emphasize that the concept applies more broadly, such as to the simulation of open quantum systems, or to more abstract differential equations.
+where the matrix exponential $U(t)=e^{-i H t}$ is known as the unitary time-evolution operator. Although this article focuses on generators of this form, it's important to emphasize that the concept applies more broadly, such as to the simulation of open quantum systems, or to more abstract differential equations.
 
-A primary goal of dynamical simulation is to implement the time-evolution operator on some quantum state encoded in qubits of a quantum computer.  In many cases, the Hamiltonian may be broken into a sum of some $d$ simpler terms
+A primary goal of dynamical quantum simulation is to implement the time-evolution operator on some quantum state encoded in qubits of a quantum computer.  In many cases, the Hamiltonian may be broken into a sum of some $d$ simpler terms
 
 $$
 \begin{align}
     H & = \sum^{d-1}_{j=0} H_j,
 \end{align}
 $$
 
-where time-evolution by each term alone is easy to implement on a quantum computer. For instance, if $H_j$ is a Pauli $X_1X_2$ operator acting on the 1st and 2nd elements of the qubit register `qubits`, time-evolution by it for any time $t$ may be implemented simply by calling the operation `Exp([PauliX,PauliX], t, qubits[1..2])`, which has signature `((Pauli[], Double, Qubit[]) => Unit is Adj + Ctl)`. As discussed later in Hamiltonian Simulation, one solution then is to approximate time-evolution by $H$ with a sequence of simpler operations
+where time-evolution by each term alone is easy to implement on a quantum computer. For example, if $H_j$ is a Pauli $X_1X_2$ operator acting on the first and second elements of the qubit register `qubits`, time-evolution by it for any time $t$ may be implemented simply by calling the operation `Exp([PauliX,PauliX], t, qubits[1..2])`, which has signature `((Pauli[], Double, Qubit[]) => Unit is Adj + Ctl)`. As discussed in [Simulating Hamiltonian Dynamics](xref:microsoft.quantum.chemistry.concepts.simulationalgorithms), one solution is to approximate time-evolution by $H$ with a sequence of simpler operations
```

Comment:
Similarly here, I'd suggest generalizing the scope of the linked content to not convey that simulation is limited to chemistry. While that's a critically important special case, there's also very important applications outside of chemistry.